src/main/resources/application.yml

```diff
@@ -44,7 +44,7 @@ management:
 soap:
   pagopa-payments:
     wsdl-base-url: "\${PAGOPA_PAYMENTS_WSDL_BASE_URL:http://localhost:8080}"
-
+    node-for-pa-url: "\${APP_NODE_FOR_PA_URL:}"
```

Comment:
let's also create the PR to set this environment variable on aks-deploy project:

This is the URL for DEV, but you should update also UAT and PROD envs
https://github.com/pagopa/p4pa-payhub-deploy-aks/blob/main/helm/dev/top/p4pa-pagopa-payments/values.yaml